api/v1/helpers/verbs/doFind.js

```diff
@@ -128,7 +128,8 @@ module.exports = function doFind(req, res, next, props) {
     redisCache.get(props.cacheKey, (cacheErr, reply) => {
       if (cacheErr || !reply) {
         // if err or no reply, get resuls from db and set redis cache
-        doFindResponse({ req, res, next }, props, opts, props.cacheKey, props.cacheExpiry);
+        doFindResponse({ req, res, next }, props, opts, props.cacheKey,
```

Comment:
do we have to pass props.cacheKey and props.cacheExpiry separately given that the function is already getting the whole props object?

Reply:
I just formatted this line as it was going over 80 characters.